kahuna/public/js/edits/image-editor.js

```diff
@@ -73,6 +75,18 @@ imageEditor.controller('ImageEditorCtrl',
         ctrl.error = true;
     }
 
+    mediaApi.canDelete(ctrl.image).then(deletable => {
+        ctrl.canBeDeleted = deletable;
+    });
+
+
+    ctrl.onDeleteSuccess = () => {
+        $state.go('upload', {}, {reload: true});
```

Comment:
After @kenoir  and I worked on the image result list with this - it was confusing.
Hence the push to move to the [stream thing](https://github.com/guardian/grid/pull/1022).

As we can explicitly tie the list and edits, I think we should do it that way (have the image list listen to edits somehow, not on a global handler).